js/card.js

```diff
@@ -0,0 +1,12 @@
+const visitingCardData= sessionStorage.getItem("visiting-card-data");
+visitingCardData = JSON.parse(visitingCardData);
+document.getElementById('first').innerHTML = visitingCardData.first_name;
+document.getElementById('last').innerHTML = visitingCardData.last_name;
+document.getElementById('title').innerHTML = visitingCardData.title ? visitingCardData.title : '';
+document.getElementById('gmail').innerHTML = visitingCardData.email;
+document.getElementById('phone-number').innerHTML = visitingCardData.phone_number;
+document.getElementById('whatsapp').href = 'https://wa.me/' + visitingCardData.phone_number;
```

Comment:
The phone number may not be the same as the Whatsapp number. Keep a separate field for the same.